mimic/imimic.py

```diff
@@ -9,7 +9,8 @@
 
 class IAPIMock(Interface):
     """
-    An :obj:`IAPIMock` provides an API.
+    An :obj:`IAPIMock` provides an API via an entry in a tenant's service
+    catalog.
```

Comment:
This is a good point; I think I'll have a PR for a final pass before we open up 3rd-party plugins, and this renaming should be part of it.